server/aws-lsp-partiql/src/server/language-server.test.ts

```diff
@@ -122,3 +123,72 @@ describe('PartiQL Language Service - Semantic Tokens', () => {
         jest.restoreAllMocks()
     })
 })
+
+describe('PartiQL Server - Hover Functionality', () => {
```

Comment:
Should we also add tests for the signature functionality for this file?

Reply:
Definitely! Will add the test for signature request handler before turning it ready to review.